README.md

```diff
@@ -65,6 +66,20 @@ Role Variables
 | glance_image            | UNDEF (mandatory if qcow_url is not used)               | This parameter specifies the name of disk in glance provider to be imported as template. |
 | template_prerequisites_tasks | UNDEF | Works only with qcow image. Specify a path to Ansible tasks file, which should be executed on virtual machine before creating a template from it. Note that qcow image must contain guest agent which reports IP address. |
 
+The `template_disks` List of dictionaries can contain following attributes:
+
+| Name               | Default value  |                                              |
+|--------------------|----------------|----------------------------------------------|
+| size               | UNDEF (Required)          | The size of the additional disk. |
+| name               | UNDEF (Required)          | The name of the additional disk.  |
+the disk will be looked up by id and will update name of the disk if it differs from the name passed in name parameter. |
```

Comment:
`The disk will be looked up by id and its name will be updated if it differs from the name passed in name parameter`